README.md

```diff
@@ -24,6 +24,8 @@ We use issues in this repo to track work. If you'd like to suggest a new topic o
 
 The software development industry is ever-changing, and our guide is a living document. Please suggest edits or changes via pull request.
 
+Getting new practices into the guide is pretty light on process. Feel free to raise a topic in Slack or at a guild meeting and drive to some consensus. Once you've done that, document your findings, submit a PR, and ask for the guild leads for review. If you think a proposal might be controversial after getting some consensus prior, you can also post the draft PR to #dev (and elsewhere if you don’t think target audience is in that channel) for feedback.
```

Comment:
@KendrickDaniel do you think we should include our github handles so it's easy for folks to tag us? i think it would be nice but i wanted to ask before doing so!

Reply:
One thought: Like with the other handbooks, it would be good to allow permissive guide changes (i.e., not necessarily funneling every change to a guild lead, but merely requiring peer review). It removes strict editorial control, but there's not been a problem trusting 18F/TTS folks to do the right thing.

Reply:
i like it!